docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,17 +1,104 @@
+
+To enable Cody Analytics:
+
+- Create an account on [Sourcegraph Accounts](https://accounts.sourcegraph.com/)
+- A user already having an account on Sourcegraph.com gets automatically migrated to Sourcegraph Accounts. Users can sign in to Cody Analytics using their email and password
+- Users without a Sourcegraph.com account, please get in touch with support to request the creation of a new account. Once the account is set up, users will be able to sign in using their email and password
+- Map your user account to a Sourcegraph instance, and this gives you access to Cody's analytics
+
+### Use StarCoder for Autocomplete
+
+When tested with other coder models for the autocomplete use case, [StarCoder](https://huggingface.co/blog/starcoder) offered significant improvements in quality and latency compared to our control groups for users on Sourcegraph.com. You can read more about the improvements in our [October 2023 release notes](https://sourcegraph.com/blog/feature-release-october-2023) and the [GA release notes](https://sourcegraph.com/blog/cody-is-generally-available).
+
+Since Sourcegraph 5.3, every Cody Enterprise setup that uses [Cody Gateway](/cody/core-concepts/cody-gateway) can opt-in to using StarCoder as an alternative to Claude Instant for autocomplete. To ensure a fast and reliable experience, we are partnering with [Fireworks](https://fireworks.ai/) and have set up a dedicated dedicated hardware deployment for our Enterprise users.
+
+To enable StarCoder go to **Site admin > Site configuration** (`/site-admin/configuration`) and change the `completionModel`:
+
+```json
+{
+  // [...]
+  "cody.enabled": true,
+  "completions": {
+    "provider": "sourcegraph",
+    "completionModel": "fireworks/starcoder"
+  }
+}
+```
+
+Cody Extension will automatically pick up this change when they connect to your Enterprise instance.
+
+It's recommended that every instance admin not using a third-party LLM provider makes this change and we are planning to make this the default in a future release.
+
+### Multi-repo Context
+
+<Callout type= "info">Multi-repo context for Sourcegraph Enterprise is supported with VS Code and JetBrains IDEs extensions.</Callout>
+
+Cody Enterprise allows up to 10 repos to support multi-repo context for users on IDE extensions. These can be added from the enhanced context selector window. Multi-repo context can search and retrieve context from multiple repositories responding to user questions, opening up more complex use cases. Enterprise users can ask questions requiring broader knowledge from across a codebase.
+
+### Cody ignore
+
+Enterprise instances can flag repositories and files that they don't want to be used by Cody as context for autocomplete and chat, safeguarding sensitive information from being returned to users.
```

Comment:
We do not allow marking repositories as unavailable for Cody.  The experimental support is only to exclude some/all files within a repo based on the ignore file at `.cody/ignore`